TodayExtension/tr.lproj/MainInterface.strings

```diff
@@ -1,14 +1,15 @@
-/* Class = "UIButton"; normalTitle = "setup wallet"; ObjectID = "fgE-Q8-uG8"; */
-"fgE-Q8-uG8.normalTitle" = "**setup wallet**";
-
-/* Class = "UILabel"; text = "Label"; ObjectID = "gQm-B2-ZXE"; */
-"gQm-B2-ZXE.text" = "**Label**";
 
 /* Class = "UILabel"; text = "scan QR code"; ObjectID = "JgG-Ah-UZk"; */
-"JgG-Ah-UZk.text" = "**scan QR code**";
+"JgG-Ah-UZk.text" = "scan QR code";
 
 /* Class = "UILabel"; text = "receive"; ObjectID = "KJN-zr-IQB"; */
-"KJN-zr-IQB.text" = "**receive**";
+"KJN-zr-IQB.text" = "receive";
 
 /* Class = "UILabel"; text = "send"; ObjectID = "QMM-JV-cTG"; */
-"QMM-JV-cTG.text" = "**send**";
+"QMM-JV-cTG.text" = "send";
```

Comment:
@JasperHeist All changes to this file  should be reverted as we haven't supported localization for the Today Ext.  (someday)